### ISSUE TITLE
Xss problem

### ISSUE THREAD

**Issue by jack-cn** [zeroclipboard/zeroclipboard#693]
when set src to: 

> ZeroClipboard.swf?id=\"))}catch(e){}if(!self.a)self.a=!prompt(document.cookie)//&width&height

Will prompt a window (should be xss), 
please refer the attached, ths.

![123](https://user-images.githubusercontent.com/35799642/35382214-792f045a-01f9-11e8-82c9-077fb5dff4d9.png)

**Comment by JamesMGreene** [comment]
Please provide the results of the following command:

```js
console.log( JSON.stringify( ZeroClipboard.state(), null, 2 ) );
```

**Comment by jack-cn** [comment]
@JamesMGreene  
It is accessed from browser address bar directly, not from html embed/object, 
so it should have not ZeroClipboard object.

**Comment by JamesMGreene** [comment]
My main question being: what version of the ZeroClipboard SWF are you using?  The old ones definitely had this vulnerability but I thought we eradicated it a long time ago.  If not, though, we should definitely address it. 😖

**Comment by jack-cn** [comment]
@JamesMGreene 
Please refer the following.
![zero_clipboard](https://user-images.githubusercontent.com/35799642/35424361-1c6a4e4c-028e-11e8-82e6-cbf250d5e785.png)
[ZeroClipboard.zip](https://github.com/zeroclipboard/zeroclipboard/files/1666537/ZeroClipboard.zip)

**Comment by JamesMGreene** [comment]
This was fixed in version `v1.0.7` almost 6 years ago (released June 8, 2012).  Please upgrade to a newer version.  Consider the latest `v1.x` (`v1.3.5`, release March 21, 2014), if not the new API in the `v2.x` series.

**Comment by jack-cn** [comment]
@JamesMGreene ths, i will try.
